charts/seqr/templates/check-for-new-samples-from-pipeline.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "seqr.fullname" . }}-check-new-samples-job
+  labels:
+    {{- include "seqr.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
```

Comment:
I couldn't figure out how to tie the `post-install` to the `hail-search` `Deployment` itself.  So... this `Job` will be triggered when we release `seqr` as well.

Reply:
yeah, unfortunately, i think the way helm handles these (and, the way ArgoCD maps them back onto Argo hooks) are chart/deployment-wide hooks. This is probably as close as you'll get without creating a separate deployment for this job.

Reply:
to confirm though, it will be triggered both on seqr releases and hail-search releases, not just seqr releases?